.azure-pipelines/mgmt.yml

```diff
@@ -4,6 +4,10 @@ pr:
     - master
     - search-preview
     - BuildToolsForSdk
+  paths:
+    exclude:
```

Comment:
This is only going to help temporarily as @chidozieononiwu is soon going to move the mgmt libraries under the sdk folder as well.

Reply:
Yep. I'm thinking when we do this we might get really specific with the path filters?